src/components/Calendar/readme.md

```diff
@@ -54,3 +54,42 @@ const calendarContainerStyles = {
     </div>
 </div>;
 ```
+
+##### Calendar with localization:
+
+```js
+initialState = {
+    date: new Date(),
+    value: { name: 'en-US', label: 'English (US)' },
+};
```

Comment:
remove this example and use the language selector in the first example, also add a margin to separate from the avatar